src/gatsby/models.js

```diff
@@ -1,9 +1,8 @@
 const markdownContent = require('./models/markdown-content')
-const docs = require('./models/docs')
 const imageSourcePaths = require('./models/image-source-paths')
 const github = require('./models/github')
 const pruneCache = require('./models/prune-cache')
 
-const models = [markdownContent, docs, imageSourcePaths, github, pruneCache]
+const models = [markdownContent, imageSourcePaths, github, pruneCache]
```

Comment:
I think `markdownContent` can be removed as well, since nothing aside from docs used it as cml.dev has no blog